src/Network/Network/Az.Network.psd1

```diff
@@ -395,7 +395,12 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'Get-AzExpressRouteCrossConnectionRouteTable', 
                'Get-AzExpressRouteCrossConnectionRouteTableSummary', 
                'Get-AzNatGateway', 'New-AzNatGateway', 'Remove-AzNatGateway', 
-               'Set-AzNatGateway'
+               'Set-AzNatGateway',
+			   'New-AzPrivateEndpoint', 'Get-AzPrivateEndpoint', 'Remove-AzPrivateEndpoint',
```

Comment:
indentation

Reply:
I cannot identify it from Visual Studio , even I delete space and enter, still same.

Reply:
Fixed. Thank you.